### ISSUE TITLE
support file-input element

### ISSUE THREAD

**Issue by eximius313** [PolymerElements/iron-form#84]
There is a very nice ```file-input``` custom element https://github.com/rnicholus/file-input but unfortunately ```iron-form``` seems to ignore it while submission.
Could you please support it?

**Comment by notwaldorf** [comment]
At the moment, `iron-form` can only track custom elements that use the `IronFormElementBehavior`, which fires a specific event when the element is attached.

**Comment by eximius313** [comment]
Is it possible to enhance third-party custom element (like ```file-input```) with ```IronFormElementBehavior``` from the "outside" of the element?

**Comment by ahedderich** [comment]
You can use the pre-submit hook the change your requests contentType and handle file inputs. See #54

**Comment by eximius313** [comment]
yes, but this is just a workaround. This issue's purpose is to ask for supporting it out of the box
